src/web/screens/apiKeyManagement.tsx

```diff
@@ -35,6 +40,17 @@ function ApiKeyManagement() {
     }
   };
 
+  const onKeyEdit: OnApiKeyEdit = async (form, setApiKey, participantId?) => {
+    try {
+      await EditApiKey(form, participantId);
+      setApiKey(await GetParticipantApiKey(form.keyId, participantId));
+      setStatusPopup({ message: 'Key succesfully edited', type: 'Success' });
```

Comment:
Maybe something like `'Your key has been updated'` would be a nicer message here? The popup already says 'Success'